utils/src/com/cloud/utils/StringUtils.java

```diff
@@ -52,7 +52,7 @@ public static boolean isUtf8Supported() {
         return Charset.isSupported(UTF8);
     }
 
-    public static Charset getDefaultCharset() {
+    protected static Charset getDefaultCharset() {
```

Comment:
looking at the history of it, i thought it was introduced only to help in testing. Also, i didnt see any reason why we should use defaultCharset(which anway is easy to get). We should be using preferred charset everywhere. I would say, lets make it public when we see a need.